### PR TITLE
Migrate examples to Core

### PR DESCRIPTION
**Pull request by aspeddro**

### CODE REVIEW

File: misc_docs/syntax/operators_pipe.mdx [hunk 1]
```diff
@@ -46,28 +46,29 @@ You can also explicitly define the argument position of a piped value by using t
...
 
 // Here, we put the result of toUpperCase into the last position
 // denoted with an _
-Js.String2.toUpperCase("example message")->logMsg(user, datestr, _)
+String.toUpperCase("example message")->logMsg(user, datestr, _)
 ```
 
 ```js
-function logMsg(user, datestr, msg) {
+unction logMsg(user, datestr, msg) {
```
Comment (cknitt) [inline misc_docs/syntax/operators_pipe.mdx:61]:
```suggestion
function logMsg(user, datestr, msg) {
```

File: misc_docs/syntax/operators_pipe.mdx [hunk 2]
```diff
@@ -46,28 +46,29 @@ You can also explicitly define the argument position of a piped value by using t
...
 }
 
 var datestr = "01-01-2021";
 
 var user = "admin";
 
-logMsg(user, datestr, "example message".toUpperCase());
+((function (__x) {
+        logMsg(user, datestr, __x);
+      })("example message".toUpperCase()));
```
Comment (cknitt) [inline misc_docs/syntax/operators_pipe.mdx:71]: This change in the JS output seems weird.

File: misc_docs/syntax/operators_triangle_pipe.mdx [hunk 3]
```diff
@@ -16,26 +16,26 @@ The `|>` operator passes a value to a function as its last argument.
...
-  return Js_math.random_int(1, size);
+  return Core__Math.Int.random(1, size);
 }
 
 function dieRollMessage(name, value) {
   return "Hi " + name + ", you rolled a " + value.toString();
 }
 
-var message = dieRollMessage("Jeremy", Js_math.random_int(1, 6));
+var message = dieRollMessage("Jeremy", dieRoll(6));
```
Comment (cknitt) [inline misc_docs/syntax/operators_triangle_pipe.mdx:38]: Here it seems we lost some inlining in v11 / uncurried.

**Review by cknitt — CHANGES_REQUESTED** [review]
Thanks a lot for your work @aspeddro!

Could you fix that `unction` -> `function` thing and then we can merge?

My other two remarks seem to be regressions in the compiler output. /cc @cristianoc

**Review by fhammerschmidt — APPROVED** [review]

**Pull request merged**